.travis.yml

```diff
@@ -1,6 +1,6 @@
 env:
   global:
-  - RELEASE_VERSION: v0.17.2
+  - RELEASE_VERSION: v0.19.4
```

Comment:
Using the matching version from the base image results in the bundles showing up as invalid when using a multi-version CRD. Using the latest non-1.x version seems to resolve that issue in Travis-CI and locally when building.